_data/chains/eip155-314.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Filecoin — Mainnet",
+  "chain": "FIL",
+  "status": "incubating",
+  "rpc": [],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "FIL",
+    "symbol": "FIL",
+    "decimals": 18
+  },
+  "infoURL": "https://filecoin.io",
+  "shortName": "filecoin",
```

Comment:
isn't shortname "fil"?

Reply:
I've never seen the Filecoin _network_ addressed as purely "fil".

Reply:
Just basing this on https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-1.json#L18 but can ask Boris about this tomorrow